### PR TITLE
Fix expected error count

### PR DESCRIPTION
**Pull request by sumengwang**

### CODE REVIEW

File: cypress/integration/plugins.validation.spec.ts [hunk 1]
```diff
@@ -93,7 +93,7 @@ describe('Plugin properties', () => {
 
     // Testing for errors found
     cy.get(dataCy(VALIDATE_BTN_SELECTOR)).click();
-    cy.get(dataCy('plugin-properties-errors-found')).should('have.text', '3 errors found.');
+    cy.get(dataCy('plugin-properties-errors-found')).should('have.text', '2 errors found.');
```
Comment (radhikav1) [inline cypress/integration/plugins.validation.spec.ts:96]: Relaying on number might be always unpredictable I think, instead can't we just check with 'errors found'  string alone?

**Review by radhikav1 — APPROVED** [review]

**Review by mrahanjam — APPROVED** [review]

File: cypress/integration/plugins.validation.spec.ts (hunk 1 above)
Comment (sumengwang) [reply cypress/integration/plugins.validation.spec.ts:96]: yea that's what I was thinking too. good point

Comment (mrahanjam) [reply cypress/integration/plugins.validation.spec.ts:96]: I'd argue that in this case number of failures matter. Looks like the tests is based on missing mandatory props. If we only rely on the text, we might miss a missing required prop when validating (e.g., expecting 3 failures while getting 2).

**Review by de-lan — APPROVED** [review]

File: cypress/integration/plugins.validation.spec.ts (hunk 1 above)
Comment (sumengwang) [reply cypress/integration/plugins.validation.spec.ts:96]: so does it make sense if the plugin changes its mandatory fields, we have to maintain the test on the ui side?

Comment (mrahanjam) [reply cypress/integration/plugins.validation.spec.ts:96]: It's not unusual for tests to have to be updated based on new changes.

**Pull request merged**